kits/sporting-director/.env.example

```
@@ -0,0 +1,4 @@
+LAMATIC_API_KEY=your_lamatic_api_key_here
+LAMATIC_PROJECT_ID=11727d21-0009-4e68-a4f3-c891a54f1baf
+LAMATIC_ENDPOINT=https://kishoharsorganization282-kishoharsproject403.lamatic.dev/graphql
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_ENDPOINT key should go before the LAMATIC_PROJECT_ID key

(UnorderedKey)